XRTK-Core/Packages/com.xrtk.core/Definitions/MixedRealityServiceConfiguration.cs

```diff
@@ -84,15 +97,48 @@ public uint Priority
             internal set => priority = value;
         }
 
-        [EnumFlags]
         [SerializeField]
-        private SupportedPlatforms runtimePlatform;
+        private RuntimePlatformEntry platformEntries = new RuntimePlatformEntry();
+
+        [NonSerialized]
```

Comment:
Private fields are non serialized by default, aren't they?

Reply:
Correct, but the object value doesn't get cleared when the editor recompiles and leaves the last object value from the prev domain reload event. Adding the `NonSerialized` attribute here makes sure that value is cleared on each domain reload.